pkg/kn/plugin/manager_test.go

```diff
@@ -100,6 +116,38 @@ func TestPluginInPath(t *testing.T) {
 	assert.Assert(t, plugin == nil)
 }
 
+func TestFindPluginInternally(t *testing.T) {
+	ctx := setup(t)
```

Comment:
Besides testing a simple case with only internal plugins, I think we could add a complicated test case with both internal plugins and external plugins to list, either to enhance the test of `ListPlugins` or the test of `ListPluginsForCommandGroup` for this complicated test case.

Reply:
Good idea, let me add this.